Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator.swift

```diff
@@ -307,7 +298,7 @@ public actor SwiftSDKGenerator {
       try await Shell.run("ar -x \(debFile)", currentDirectory: tmp, shouldLogCommands: isVerbose)
 
       try await Shell.run(
-        "PATH='/bin:/usr/bin:\(Self.homebrewPrefix)/bin' tar -xf \(tmp)/data.tar.*",
+        "tar -xf \(tmp)/data.tar.*",
```

Comment:
what if `tmp` has spaces? Can we not use shell stuff here?

Reply:
I agree, we should switch to `AsyncProcess` instead of `Shell` and use `-C` option on `tar` to work around the lack of `currentDirectory` property on `AsyncProcess`.

Reply:
That should be done in a separate PR though, as that's unrelated to `PATH` changes.

Reply:
Agreed.   The SDK builder shells out in several places so a separate PR to make some improvements to the underlying infrastructure would be good.